pkg/kubelet/dockershim/docker_service.go

```diff
@@ -135,6 +137,21 @@ func NewDockerService(client dockertools.DockerInterface, seccompProfileRoot str
 	}
 	ds.networkPlugin = plug
 	glog.Infof("Docker cri networking managed by %v", plug.Name())
+
+	// NOTE: cgroup driver is only detectable in docker 1.11+
+	var cgroupDriver string
+	dockerInfo, err := ds.client.Info()
+	if err != nil {
+		return nil, fmt.Errorf("Failed to execute Info() call to the Docker client: %v", err)
```

Comment:
errors should start in lower form.

i think if we are unable to make the client call, we can default to cgroupfs.